test/__snapshots__/index.spec.ts.snap

```diff
@@ -7565,7 +7567,10 @@ export abstract class MultipartService {
         return __request(this.client, this.config, options || {}, {
             method: 'POST',
             url: '/api/v{api-version}/multipart',
-            formData: formData,
+            formData: {
+                'content': data?.content,
```

Comment:
What is `content`?

Reply:
Is it the content type?

Reply:
No, this is the name of a property. So the multipart request has a property called `content` which we now use instead of using the name of the ref.

Reply:
We cannot just use `data` here because `data` holds both query parameters and body parameters, so we need to be specific and tie things correctly. This is the same treatment we do on json content types.

Reply:
ok ok